src/main.rs

```diff
@@ -142,13 +161,21 @@ fn main() {
                 let mut commands = renderer
                     .device
                     .create_command_encoder(&wgpu::CommandEncoderDescriptor { label: None });
-                    renderer.render(&mut commands, &frame.view, &rendering::defaults::axes(), &rendering::defaults::render_all_vertices(&rendering::defaults::axes()), true);
-                renderer.render(&mut commands, &frame.view, &vertices, &indecies, false);
+                renderer.render(&mut commands, &frame.view, &rendering::defaults::axes(), &rendering::defaults::render_all_vertices(&rendering::defaults::axes()), true);
+                //renderer.render(&mut commands, &frame.view, &vertices, &indecies, false);
+                for i in 0..lines.len() {
+                    let v = &lines[i].verticies;
+                    let i = &lines[i].indicies;
+                    renderer.render(&mut commands, &frame.view, v, &i, false);
```

Comment:
This is going to render individual points on the line as unconnected dots; is that your intent?

(And if it is, do you want to eventually support the connections? If so, make an issue 😃 )

Reply:
done. https://github.com/dr-sg/rscat/issues/12